.github/workflows/publish-to-pypi.yml

```diff
@@ -20,11 +20,9 @@ jobs:
         python3 -m pip install setuptools wheel twine
         pip3 install -r requirements.txt
     - name: Run linter
-      run: python3 -m pylint wheatley/**/*.py
-    - name: Run tests
-      run: python3 -m pytest
-    - name: Run fuzzer
-      run: ./fuzz
+      run: python3 run-checks.py --lint
+    - name: Run unit tests
+      run: python3 run-checks.py --unit-test
```

Comment:
Chance to consider which check should run when publishing to pip.
They'll all should have been run as part of `main.yml` when it was merged, so long as we don't rush and ignore a failure there.

- Unit tests are quick and deterministic.
- If we lint, should we also type check?
- Doc tests make web requests, so can fail based on external resources.
- Fuzzing is random inputs

Reply:
Yeah I think we should do unit tests, linting and typechecking.  Fuzzing is fast but technically involves randomness (and anyway it's not the end of the world if Wheatley outputs janky error messages sometimes when parsing), but doc tests are too temperamental.